### DISCUSSION TITLE
docker-compose up take too much times

### DISCUSSION THREAD

**Discussion by AnythingLinux** [saleor/saleor#11266] [Deployment]
I am using Ubuntu 22.04. I am using saleor platform. After i run docker compose up its take too much times.

**Comment by CossackDex** [comment]
Hi @UbuntuEvangelist, 
Without more context, we are not able to help you. Have you considered using our Cloud solution instead of local development? You can create a free developer account for testing purpose using CLI: https://github.com/saleor/saleor-cli
